tests/parser/test_parser.py

```diff
@@ -18,7 +18,7 @@
 )
 from pyparsing import ParseException
 
-default_parse = partial(parse, default_sampling_method=SamplingMethod.DEFAULT)
+default_parse = partial(parse)
```

Comment:
Is this still needed?

Reply:
Not necessarily, I just wanted to keep the diff a bit shorter.